staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -295,7 +302,41 @@ def update(self):
         self.updatePlaylistDetails()
         self.updateCoverArtFromURLs()
 
-  ##############################################################################
+        #trigger a hdmi command, if a controller has changed the power status or the volume
+        if constants.TURNONAVR == 'true' and constants.CECSUPPORT == 'true':
+          powerstate = self.player.getpowerstate()
+          if powerstate == 0 and constants.HDMIPOWER == "true":
+            if constants.TURNOFFTV == "true":
+               tv.standby()
+               pass
+            if constants.TURNONAVR == "true":
+               avr.standby()
+            constants.HDMIPOWER = "false"
```

Comment:
It should be a class vairable in NowPlayingWindow.py most likely.  The main thing is if you define a constant, never assign to it.  It's a shame in fact python doesn't have a constant declaration to prevent this...you can sort of hack them in by making mini fucntions..

def MY_CONST_VALUE(): return 123 

...but that's a lot of waste.  make it a class variable (only one NPW is made for each run of Xsqueeze and it hangs around till exit)